### PR TITLE
Add docs to demo how to deploy and use triton inference on alibaba cloud's PAI-EAS

### PR DESCRIPTION
**Pull request by Orion34C**
Hi, we are from alibaba cloud computing's PAI team. Recently we have supported triton inference server on PAI-EAS. We have prepared the doc in this pull request to help users deploy triton inference server on EAS.

### CODE REVIEW

File: deploy/alibaba-cloud/README.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+strict-model-config
+strict-readiness
+allow-http
+http-thread-count
+pinned-memory-pool-byte-size
+cuda-memory-pool-byte-size
+min-supported-compute-capability
+buffer-manager-thread-count
+backend-config
+```
```
Comment (CoderHam) [inline deploy/alibaba-cloud/README.md:151]: Was `model-repository` omitted intentionally?

File: deploy/alibaba-cloud/README.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
+# Deploy Triton Inference Server on PAI-EAS 
+* Table Of Contents
+   - [Description](https://yuque.alibaba-inc.com/pai/blade/mtptqc#Description)
+   - [Prerequisites](https://yuque.alibaba-inc.com/pai/blade/mtptqc#Prerequisites)
+   - [Demo Instruction](https://yuque.alibaba-inc.com/pai/blade/mtptqc#31bb94ef)
+   - [Additional Resources](https://yuque.alibaba-inc.com/pai/blade/mtptqc#89d5e680)
+   - [Known Issues](https://yuque.alibaba-inc.com/pai/blade/mtptqc#558ab0be)
+
+# Description
+This repository contains infomation about how to deploy NVIDIA Triton Inference Server in  EAS(Elastic Algorithm Service) of Alibaba-Cloud.
```
Comment (CoderHam) [inline deploy/alibaba-cloud/README.md:10]: infomation -> information

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: deploy/alibaba-cloud/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md]: done

File: deploy/alibaba-cloud/README.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- EAS providing a simple way for deep learning developers to deploy their models in Alibaba Cloud.
```
[the remark this answers is not in the document]
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md]: done

File: deploy/alibaba-cloud/README.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+# Additional Resources
+See the following resources to learn more about how to use Alibaba Cloud's OSS orEAS.
+- [Alibaba Cloud OSS's Document](https://help.aliyun.com/product/31815.html?spm=a2c4g.11186623.6.540.3c0f62e7q3jw8b)
+
+
+# Known Issues
+- [Binary Tensor Data Extension](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md) is not fully supported yet, users want to use service with binary extension supported, it is only available in cn-shanghai region of PAI-EAS.
+- Currently only HTTP/1 is supported, hence gRPC cannot be used when query Triton servers on EAS. HTP/2 will be officially supported in a short time.
+- Users should not mount a whole OSS bucket when launching Triton processor, but an arbitrarily deep sub-directory in bucket. Otherwise the mounted path will no be as expected.
+- Not all of Triton Server parameters are be supported on EAS, the following params are the ones supported now.
```
[the remark this answers is not in the document]
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md]: done

[the remark this answers is not in the document]
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md]: done

File: deploy/alibaba-cloud/README.md (hunk 2 above)
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md]: done

File: deploy/alibaba-cloud/README.md (hunk 1 above)
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md:152]: It's not intentionally, I add it now.

Comment (CoderHam) [reply deploy/alibaba-cloud/README.md:152]: I do not see model-repository yet

**Review by CoderHam — CHANGES_REQUESTED** [review]

File: deploy/alibaba-cloud/README.md (hunk 1 above)
Comment (Orion34C) [reply deploy/alibaba-cloud/README.md:152]: It is in line 137 of current file.

**Review by CoderHam — DISMISSED** [review]

**Comment by Orion34C** [comment]
> To avoid potential copyright issues can you remove cat.png from the change and instead change the instructions to point to a publically available cat image or give instructions on how to download a cat image.

The origin image has been deleted. Add a simple comment in the python script.

**Review by CoderHam — APPROVED** [review]

**Comment by CoderHam** [comment]
@deadeyegoodwin I am merging the same since the user made the changes you suggested.

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**